app-ios/Modules/Sources/Theme/Typography.swift

```diff
@@ -0,0 +1,47 @@
+import SwiftUI
+
+struct Typography {
+    var displayLarge: TextStyle = TypographyTokens.DisplayLarge
+    var displayMedium: TextStyle = TypographyTokens.DisplayMedium
+    var displaySmall: TextStyle = TypographyTokens.DisplaySmall
+    var headlineLarge: TextStyle = TypographyTokens.HeadlineLarge
+    var headlineMedium: TextStyle = TypographyTokens.HeadlineMedium
+    var headlineSmall: TextStyle = TypographyTokens.HeadlineSmall
+    var titleLarge: TextStyle = TypographyTokens.TitleLarge
+    var titleMedium: TextStyle = TypographyTokens.TitleMedium
+    var titleSmall: TextStyle = TypographyTokens.TitleSmall
+    var bodyLarge: TextStyle = TypographyTokens.BodyLarge
+    var bodyMedium: TextStyle = TypographyTokens.BodyMedium
+    var bodySmall: TextStyle = TypographyTokens.BodySmall
+    var labelLarge: TextStyle = TypographyTokens.LabelLarge
+    var labelMedium: TextStyle = TypographyTokens.LabelMedium
+    var labelSmall: TextStyle = TypographyTokens.LabelSmall
+}
+
+struct TextStyle {
+    var fontSize: CGFloat
+    var fontFamily: String
+    var fontWeight: SwiftUI.Font.Weight
+    var fontStyle: SwiftUI.Font
+    var lineHeight: CGFloat
+    var letterSpacing: CGFloat?
+}
```

Comment:
I think these struct and variables should be `public` accessible level for access from other modules 👀 
If my understanding is correct, please add `public` accessor 🙏🏼